R-package/demo/basic_walkthrough.R

```diff
@@ -55,6 +55,8 @@ print(paste("test-error=", err))
 # save model to binary local file
 xgb.save(bst, "xgboost.model")
 # load binary model to R
+# Function doesn't take 'nthreads', but can be set like this:
+RhpcBLASctl::omp_set_num_threads(1)
```

Comment:
Please note that this also modifies all the thread numbers for the rest of the demo, including prediction.

Reply:
That shouldn't be a problem, since the rest of the demo also needs to follow the same thread limitations.